cranelift/filetests/filetests/vcode/arm64/arithmetic.clif

```diff
@@ -0,0 +1,242 @@
+test vcode arch=arm64
```

Comment:
Why not use the existing `target aarch64` directive?

Reply:
The simple answer is that I couldn't work out how to fetch it. It doesn't appear to be included in the `preamble_comments` of the `Context`. It doesn't seem too important to me; but I'm happy to change it if you can point out the proper plumbing to make use of here.

Reply:
Actually, upon second look, worked this out -- the tests now use the `target` directive. Resolving!